i18n/nl/cosmic_settings.ftl

```diff
@@ -0,0 +1,807 @@
+    .auto-suspend-battery = Automatische slaapstand wanneer op batterijstroom
+
+## Input
+
+acceleration-desc = De trackinggevoeligheid wordt automatisch aangepast op basis van de snelheid.
+
+disable-while-typing = Tijdens het typen uitschakelen.
+
+input-devices = Invoerapparaten
+    .desc = Invoerapparaten
+
+primary-button = Primaire knop
+    .desc = Bepaalt de rangschikking van de fysieke toetsen.
+    .left = Links
+    .right = Rechts
+
+scrolling = Scrollen
+    .two-finger = Met twee vingers scrollen
+    .edge = Scroll met een vinger langs de rand
+    .speed = Scrollsnelheid
+    .natural = Natuurlijk scrollen
+    .natural-desc = Scroll de inhoud, niet het weergavevenster.
+
+## Input: Keyboard
+
+slow = Langzaam
+fast = Snel
+short = Kort
+long = Lang
+keyboard = Toetsenbord
+    .desc = Invoermethodes, omschakelen, invoer van speciale tekens en sneltoetsen.
+
+keyboard-sources = Invoermethodes
+    .desc = Invoermethodes kunnen worden gewisseld met de sneltoetscombinatie Super+Spatie. Dit kan aangepast worden in de instellingen voor sneltoetsen.
+    .move-up = Omhoog
+    .move-down = Naar beneden
+    .settings = Instellingen
+    .view-layout = Toetsenbordindeling weergeven
+    .remove = Verwijderen
+    .add = Invoermethode toevoegen
+
+keyboard-special-char = Invoer speciale tekens
+    .alternate = Toets voor speciale tekens
+    .compose = Compose-toets (voor speciale tekens)
+    .caps = Caps Lock-toets
+
+keyboard-typing-assist = Typen
+    .repeat-rate = Snelheid van herhalen
+    .repeat-delay = Herhalingsvertraging
+
+added = Toegevoegd
+type-to-search = Typ om te zoeken…
+show-extended-input-sources = Toon uitgebreide invoermethodes
+
+## Input: Keyboard: Shortcuts
+
+keyboard-shortcuts = Sneltoetsen
+    .desc = Sneltoetsen bekijken en aanpassen
+
+add-keybinding = Sneltoets toevoegen
+cancel = Annuleren
+command = Commando
+custom = Aangepast
+debug = Fouten opsporen
+disabled = Uitgeschakeld
+migrate-workspace-prev = Werkblad naar vorige uitvoer verplaatsen
+migrate-workspace-next = Werkblad naar volgende uitvoer verplaatsen
+migrate-workspace = Werkblad verplaatsen naar uitvoer { $direction ->
+    *[down] hieronder
+    [left] links
+    [right] rechts
+    [up] hierboven
+}
+navigate = Navigeren
+replace = Vervangen
+shortcut-name = Sneltoetsnaam
+system-controls = Systeemaansturing
+terminate = Beëindigen
+toggle-stacking = Zet vensterstapelen (stacking) aan/uit
+type-key-combination = Voer toetsencombinatie in
+
+custom-shortcuts = Aangepaste sneltoetsen
+    .add = Sneltoets toevoegen
+    .context = Aangepaste sneltoets toevoegen
+    .none = Gebruik geen aangepaste sneltoetsen
+
+modified = { $count } aangepast
+
+nav-shortcuts = Navigatie
+    .prev-output = Focus op vorige uitvoer
+    .next-output = Focus op volgende uitvoer
+    .last-workspace = Focus op laatste werkblad
+    .prev-workspace = Focus op vorige werkblad
+    .next-workspace = Focus op volgende werkblad
+
```

Comment:
Realy confusing English sentence to translate: Focus window down / in / left ... What does it mean!? 

What is meant by: focus window down /in /left ...
   What is the subject and object of the sentence?